python-sdk/tests/databases/test_snowflake.py

```diff
@@ -196,14 +199,33 @@ def test_if_exist_param_of__load_pandas_dataframe_to_table(database_table_fixtur
 
     pandas_dataframe = pd.DataFrame(data={"id": [1, 2]})
 
-    with mock.patch("snowflake.connector.pandas_tools.write_pandas"):
-        with mock.patch("astro.databases.base.BaseDatabase.create_table") as method:
-            database.load_pandas_dataframe_to_table(pandas_dataframe, table, if_exists="replace")
-            method.assert_called_with(table, dataframe=pandas_dataframe)
+    with mock.patch("snowflake.connector.pandas_tools.write_pandas") as method:
+        database.load_pandas_dataframe_to_table(pandas_dataframe, table, if_exists="replace")
+        method.assert_called_with(
+            conn=ANY,
+            df=ANY,
```

Comment:
why is `df` ANY?

shouldn't it be `df=pandas_dataframe`?

Reply:
With this test case, we are just concerned if the `auto_create_table` is true or false base on the append/replace value. I can add it, no strong opinion.